src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -356,27 +352,27 @@ class Microsoft::Terminal::Core::Terminal final :
         til::point pivot;
     };
     std::optional<SelectionAnchors> _selection;
-    bool _blockSelection;
+    bool _blockSelection = false;
     std::wstring _wordDelimiters;
-    SelectionExpansion _multiClickSelectionMode;
-    SelectionInteractionMode _selectionMode;
-    bool _selectionIsTargetingUrl;
-    SelectionEndpoint _selectionEndpoint;
-    bool _anchorInactiveSelectionEndpoint;
+    SelectionExpansion _multiClickSelectionMode = SelectionExpansion::Char;
+    SelectionInteractionMode _selectionMode = SelectionInteractionMode::None;
+    bool _selectionIsTargetingUrl = false;
+    SelectionEndpoint _selectionEndpoint = SelectionEndpoint::None;
+    bool _anchorInactiveSelectionEndpoint = false;
 #pragma endregion
 
     std::unique_ptr<TextBuffer> _mainBuffer;
     std::unique_ptr<TextBuffer> _altBuffer;

```

Comment:
```suggestion
    std::unique_ptr<TextBuffer> _mainBuffer = nullptr;
    std::unique_ptr<TextBuffer> _altBuffer = nullptr;
```
no benefit to initializing these too? At least for consistency.